src/OVAL/probes/probe/worker.c

```diff
@@ -36,6 +36,10 @@
 #include <pthread_np.h>
 #endif
 
+#if defined(OS_OSX)
```

Comment:
I wonder if this really works. And I don't think it should be a problem to include this file in all platforms.

Reply:
It probably should. Since: https://github.com/OpenSCAP/openscap/blob/maint-1.3/compat/oscap_platforms.h#L52 defines

anyway... it's still failing. You can try if without the `#if` has the same effect